cms/envs/production.py

```diff
@@ -452,6 +452,12 @@ def get_env_setting(setting):
 # Queue to use for updating grades due to grading policy change
 POLICY_CHANGE_GRADES_ROUTING_KEY = ENV_TOKENS.get('POLICY_CHANGE_GRADES_ROUTING_KEY', DEFAULT_PRIORITY_QUEUE)
 
+# Queue to use for individual learner course regrades
+SINGLE_LEARNER_COURSE_REGRADE_ROUTING_KEY = ENV_TOKENS.get(
+    'SINGLE_LEARNER_COURSE_REGRADE_ROUTING_KEY',
+    DEFAULT_PRIORITY_QUEUE
+)
```

Comment:
I think this is the right place to be like "use the default queue if nothing more specific has been provided"; I don't think it makes sense to set the default to something other than what we'll want it to be today (i.e. the single_learner_course_regrade-named queue) in edx-internal & configuration.